content/src/content/jcr_root/apps/core/wcm/components/teaser/v1/teaser/clientlibs/editor/js/teaser.js

```diff
@@ -198,4 +199,11 @@
             }
         }
     }
+
+    function associateDescriptionTextFieldWithLabel(descriptionTextfieldElement) {
+        var richTextContainerSelector = ".cq-RichText.richtext-container";
+        var $richTextContainerParent = $(richTextContainerSelector)[0].parentNode;
+        var $descriptionLabel = $richTextContainerParent.children[0];
```

Comment:
I think this is error-prone as:
- there's no check the elements actually exist
- `.children[0]` might not be the right element if the RTE structure changes